sympy/functions/elementary/trigonometric.py

```diff
@@ -397,6 +397,9 @@ def _eval_rewrite_as_sqrt(self, arg):
     def _eval_rewrite_as_csc(self, arg):
         return 1/csc(arg)
 
+    def _eval_rewrite_as_sec(self, arg):
+        return sqrt(1 - (1 / sec(arg)**2))
```

Comment:
Square roots should be avoided since their sign may be incorrect. One could probably use `1/sec(arg - S.Pi/2)`.